benchmarks/test_benchmark_indicators.py

```diff
@@ -0,0 +1,55 @@
+from stock_indicators import indicators
+
```

Comment:
It will be interesting to see the performance tests results here; and compare them to the .NET library, to see how much extra performance cost there is from the wrapper.  I'm assuming you'll add those performance tests results in your `init-docs` branch?

Reply:
> I'm assuming you'll add those performance tests results in your init-docs branch?

Right. This branch is for doing so.